bokeh/core/enums.py

```diff
@@ -93,10 +93,10 @@ def enumeration(*values, **kwargs):
 Direction = enumeration("clock", "anticlock")
 
 #: Specify units for mapping values
-Units = enumeration("screen", "data")
+Units = enumeration("screen", "data")  # TODO (bev): can this be deleted?
```

Comment:
@bryevdv, `git grep` shows no usage, besides `SpatialUnits = Units` and being imported in `test_glyphs.py`.